roles/node/files/inventory-sample.yml

```diff
@@ -0,0 +1,31 @@
+all:
```

Comment:
This is not the best place to store inventory files. You can insert this file as a code block to `roles/node/README.md`.
If you need to store it as a file ( e.g for the local test) you can put it in  `roles/node/molecule/' folder.

Reply:
Moved to README